.github/workflows/unittests.yml

```diff
@@ -0,0 +1,14 @@
+name: Run Python unittests
+
+on:
+  push:
+
+jobs:
+  deploy:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout GitHub workspace
+        uses: actions/checkout@v2
+
+      - name: Install python, activate venv, install dependencies, & run tests all in docker container
+        run: sudo docker run --rm -v $PWD:/src -w "/src" ubuntu:latest bash -c "apt update && apt -y upgrade && apt install -y python3-pip python3-venv && ./setup.sh && source ./venv/bin/activate && ./runtests.sh"
```

Comment:
To verify if it's working, you can run `docker run --rm -v $PWD:/src -w "/src" ubuntu:latest bash -c "apt update && apt -y upgrade && apt install -y python3-pip python3-venv && ./setup.sh && source ./venv/bin/activate && ./runtests.sh"` on your local machine in the project-tetra-display root directory to emulate what's happening in the CI build, or look at the recent build itself https://github.com/tetrabiodistributed/project-tetra-display/pull/7/checks?check_run_id=941830230. This PR is more about validating whether this is the automated workflow we actually want in a CI build.